packages/etcd/test/test.js

```diff
@@ -0,0 +1,37 @@
+const test = require('ava');
+const KeyvEtcd = require('this');
+
+const etcdURL = '127.0.0.1:2379';
+
```

Comment:
Can you please add in the standard test-suite into this as it is used for all testing.

```javascript
keyvOfficialTests(test, Keyv, 'mysql://root@localhost/keyv_test', 'mysql://foo');

const store = () => new KeyvMysql('mysql://root@localhost/keyv_test');
keyvTestSuite(test, Keyv, store);
```

Reply:
@jaredwray Can you help me in handling errors?  The documentation says to handle errors as specified in [link](https://microsoft.github.io/etcd3/interfaces/ioptions.html#faulthandling)

But Its not working

Reply:
When you add in the code above does it fail locally? If so, have you tried to use a try/catch around the code and if it fails to handle it correctly?

Reply:
I have fixed it with help of this https://github.com/microsoft/etcd3/issues/105